Froggie/Source/Froggie.Domain/Groups/CreateGroupService.cs

```diff
@@ -1,27 +1,38 @@
-﻿namespace Froggie.Domain.Groups;
+﻿using Froggie.Domain.Users;
+
+namespace Froggie.Domain.Groups;
 
 public interface ICreateGroupService
 {
-    ValueTask<Group> CreateAsync(string nameValue);
+    ValueTask<Group> CreateAsync(User creator, string nameValue);
+    ValueTask<Group> CreatePersonalAsync(User creator);
 }
 
 internal sealed class CreateGroupService : ICreateGroupService
 {
     private readonly IGroupFactory groupFactory;
     private readonly IAddGroupCommand addGroupCommand;
+    private readonly IAddUserToGroupService addUserToGroupService;
 
-    public CreateGroupService(IGroupFactory groupFactory, IAddGroupCommand addGroupCommand)
+    public CreateGroupService(IGroupFactory groupFactory,
+                              IAddGroupCommand addGroupCommand,
+                              IAddUserToGroupService addUserToGroupService)
     {
         this.groupFactory = groupFactory;
         this.addGroupCommand = addGroupCommand;
+        this.addUserToGroupService = addUserToGroupService;
     }
 
-    public async ValueTask<Group> CreateAsync(string nameValue)
+    public async ValueTask<Group> CreateAsync(User creator, string nameValue)

```

Comment:
One question I'd like to pose to both of us is whether or not we see a value in having a creator as a group field. I think that my gut feeling is that it IS useful to have a creator field but when I think more practically I'm not sure I actually see a use for it. Eventually we probably want to have group administrators that can send invites and reject or approve join requests and edit group settings, and I assume we'll want to be able to extend administrator privileges to other people in the group. The only thing I see creator potentially being helpful for is for Creator to be basically the same role as Administrator but the Creator can never have their Creator status revoked as there always needs to be at least one administrator